tests/containers/volumes.pm

```diff
@@ -127,11 +127,23 @@ sub run {
     test $runtime;
     $engine->cleanup_system_host();
 
-    if ($runtime eq "podman" && !is_public_cloud && !is_svirt && !is_transactional) {
-        record_info "rootless";
-        select_user_serial_terminal();
-        test $runtime;
+    # select_user_serial_terminal() is broken on these:
+    return if (is_public_cloud || is_svirt || is_transactional);
```

Comment:
I'd say it's better to have this in the `main_containers.pm` schedule as it make a visually better distinction between an empty test run and a not scheduled one.

There's always a certain danger in letting a test run "run empty", because the likelyhood that this will be spotted is lower than a test run that is not present.